package.json

```diff
@@ -11,17 +11,16 @@
     "url": "https://github.com/feross/p2p-graph/issues"
   },
   "dependencies": {
-    "d3": "^3.5.6",
-    "debounce": "^1.0.0",
-    "debug": "^2.2.0",
-    "prettier-bytes": "^1.0.3"
+    "d3": "3.5.6",
+    "debounce": "1.0.0",
+    "debug": "2.2.0",
+    "prettier-bytes": "1.0.3"
```

Comment:
Great, thanks. It's worth the risk of breaking changes to get new patch and minor versions automatically. Accidental breaking changes happen extremely rarely in my experience.